infinity-keys/ik-landing
# s

- KABOOM
- Hello html
- add redwood
- Home page
- Start of redwood on render
- yml -> yaml
- Redwood api config
- Yarn install before yarn deploy
- Run setup command
- Enable prerendering test
- Tailwind
- Redwood upgrade
- Testing if migrations are applied to render
- ENG-417: add-global-css-and-fonts (#415)
- ENG-418: add-ui-components (#416)
- ENG-421: add wrapper and related components (#418)
- conflict fix
- empty migration
- ENG-419: get-homepage-working (#419)
- ENG-420: markdown and alert components (#423)
- ENG-424: cloudinary-components (#424)
- ENG-427: email compononets (#429)
- ENG-428: grid components (#430)
- ENG-432: wallet components (#431)
- ENG-433: privacy and 404 pages (#432)
- ENG-434: cookie consent 🍪 (#434)
- ENG-439: fix linter errors (#437)
- ENG-436: skip link (#436)
- ENG-438: create user model in db (#440)
- Rename veercel.json to vercel.json (#491)
- ENG-443: profile cell and page (#448)
- ENG-445: add loading icon functionality (#446)
- Dockerize postgres for easy local db setup (#490)
- Bump postgres to 15
- Return to original db name, but as postgres 15 this time
- ENG-475: trigger reply email (#482)
- ENG-476: add delete info view and logic (#487)
- Update schema for many to many relationships (#499)
- Schema updates (#504)
- itsmekori/eng-492-update-partner-section (#500)
- update hero section with styling and success component
- Update redwood deps (#512)
- ENG-416: migrate rest routes (#520)
- ENG-516: organize sdl files (#528)
- Itsmekori/eng 506 bring over puzzle component UI elements (#519)
- Schema update for rewardable concept (#518)
- ENG-517: migrate nft check api routes (#533)
- Tawnee/eng 526 lens share package (#543)
- Tawnee/eng 527 move utils into packages (#546)
- ENG-525: new puzzles + seed (#544)
- render config api (#555)
- feature/route order (#556)
- setup from demo
- add auth page, get login with email flow working
- add save to db example
- verify rework
- making wallet age check dumber, migrating contract lookup
- check NFT cleanup
- check-claimed cleanup
- check claimed further comments
- check balance api cleanup
- comments update
- fixing gitignore issues
- update user schema
- add reddit svg to footer (#602)
- update authid
- ENG-584: hero particles (#587)
- ENG-532: replace web and api libs with packages (#549)
- add alpha holders data query (#603)
- ENG-596: update vite in packages (#604)
- replace contractLookups with lookups, update imports
- migrate files and renamed component
- update syntax  and fix prettier errors
- update reddit footer link (#607)
- minor updates and fixes, add a few todos
- delete index.js file and update padding on instructions
- add require auth functionality. get getCurrentUser working
- add public address and site role on profile creation
- add some todos
- remove public address from db
- remove public address references, add walletbutton
- use address in profilecell
- wait for mutation to finish before showing profile, add auth loading icon in header
- profile username
- get claim page returning signature
- get claim function working
- update variable name
- remove hooks in hooks
- rename file
- remove hooks, update error messages
- Eng-547: add step UI (#558)
- ENG-548 Puzzle UI (#568)
- add infura key to env
- ham/eng 591 standardizing and simplifying api routes (#600)
- Itsmekori/eng 625 create discord mint tracking bot (#622)
- push console logs (#627)
- Discord bot debug (#628)
- update env variable names (#629)
- check if tx confirmed before post (#630)
- migrate custom css, images and update code to show css
- set up styles for button and code cleanup
- move style for social button to step page
- ham/eng 478 create poap check (#540)
- itsmekori/eng-637-update-saga-packpuzzles-in-redwood (#638)
- itsmekori/eng-639-redwood-update-lens-collector (#640)
- migrate nft metadata (#641)
- Itsmekori/eng 646 add to redwood (#643)
- bug/node fetch (#645)
- itsmekori/eng-656-remove-duplicate-link (#648)
- ENG-459: implement magiclink auth (#598)
- ENG-554: Steps standalone cell (#557)
- ENG-626: save user attempts to db (#631)
- bug/env (#659)
- add packs to seed.js (#646)
- ENG-679: use sets for wrappers (#660)
- ENG-680: link nft to rewardable (#661)
- ENG-671: roles for user data (#655)
- get token ids and gated ids dynamically
- center claim text
- get data from update rewardable. get nft token ids only
- add nft check to schema, attempt, and makeattempt. add nft check button
- add step type lookups and helper function.
- get nftcheck data from db. add useMakeAttempt hook
- remove comments. move solution data to types
- enable poap checks
- make usemakeattempt lower case
- add user rewards schema and sdl
- update sdls
- create user reward on last step solve
- create userReward when all puzzles in pack are solved
- update claim service to check for completed rewardables
- add nft to userreward on successful transaction
- block user attempt on solved puzzle
- remove comment
- add some comments
- filter user rewards by user id. use rewardable type
- test pr (#668)
- remove success and add erros to web3 services. add userId to userrewards
- add comments
- remove messages. remove verify and put logic directly in claim
- rename variable. add comment
- add types
- update attempts with new logic
- update usemakeattempt
- ENG-603: Claim page and minting (#619)
- ENG-704: little fixes (#672)
- parse step type. update attempt service
- ENG-708: add-gnosis-chain-to-rw (#674)
- ENG-710: update mobile header (#675)
- auth styles (#677)
- update step type logic
- add contract addresses to readme
- use disc. union for attempt data
- remove log
- bring over nft check logic
- replace any
